BraveWallet/Crypto/Transaction Confirmations/TransactionConfirmationView.swift

```diff
@@ -71,8 +71,10 @@ struct TransactionConfirmationView: View {
   /// View showing the currently selected account with a blockie
   @ViewBuilder private var accountView: some View {
     HStack {
-      Text(keyringStore.selectedAccount.address.truncatedAddress)
-        .fontWeight(.semibold)
+      AddressView(address: keyringStore.selectedAccount.address) {
+        Text(keyringStore.selectedAccount.address.truncatedAddress)
+          .fontWeight(.semibold)
+      }
```

Comment:
Should we show the account in `TransactionHeader` too for non ERC20 approve transactions? We only show the account name currently, but I think this could be useful for the to/from on the other transaction types too

Reply:
i think we can keep this as it is since it doesnt display a truncated account address.
once we decided to add account address information we will add a context menu to align with other screens.

Reply:
If we don't know the name of the account, ex in a send transaction, we show the truncated address. I think we need context menu there.